File: PYTHON/opentutorials/Function.py
ids=['a', 'b', 'c']
input_id=input("ENTER ID\n")



def login(input):
    for id in ids:
        if id==input_id:
            return True
    return Flase



if login(input_id):
    print("hello, "+input_id)
else:
    print("who r u?")

File: RUBY/opentutorials/Function.rb
ids=['a', 'b', 'c']
puts("ENTER ID")
input_id=gets.chomp()


def login(input)
  for id in ids do
    if id==input
      return true
  end
  return false
end

if login(input_id)
  puts("hello, "+input_id)
else
  puts("who r u?")
